File: test/xunit/TestClass1.cs
using System;
using Shouldly;
using Xunit;

namespace XunitTests
{

    public class TestClass1
    {
        [Fact]
        public void Pass()
        {
            (1+1).ShouldBe(2);
        }

        [Fact]
        public void AnotherPass()
        {
            (1+1).ShouldBe(2);
        }

        [Fact]
        public void Fail()
        {
            (1+1).ShouldBe(2);
        }        
    }
}

File: test/fsxunittests/TestClass1.fs
module FSharpTests

open Xunit
open Shouldly

[<Fact>]
let ``This is a test that has spaces in it's name`` () =
   (true).ShouldBe(true)

[<Fact>]
let ``This is a test that has (parens) in it's name`` () =
   (true).ShouldBe(true)
